src/modelAnalysis/FVA/fluxVariability.m

```diff
@@ -1,7 +1,7 @@
-function [minFlux,maxFlux,Vmin,Vmax] = fluxVariability(model,optPercentage,osenseStr,rxnNameList,verbFlag, allowLoops)
+function [minFlux,maxFlux,Vmin,Vmax] = fluxVariability(model,optPercentage,osenseStr,rxnNameList,verbFlag, allowLoops, method)

```

Comment:
why do you use 2-norm as default method? and not FBA which was the default earlier?